lib/CMakeLists.txt

```diff
@@ -1,5 +1,2 @@
-
-if(ENABLE_PYTHON)
-    install(DIRECTORY ambit DESTINATION ${PYTHON_SITE_PACKAGES} FILES_MATCHING PATTERN "*.py")
-    install(FILES np_test.py test_blocks.py test_operators.py DESTINATION ${PYTHON_SITE_PACKAGES}/ambit/tests)
-endif()
+install(DIRECTORY ambit DESTINATION ${PYTHON_SITE_PACKAGES} FILES_MATCHING PATTERN "*.py")
+install(FILES np_test.py test_blocks.py test_operators.py DESTINATION ${PYTHON_SITE_PACKAGES}/ambit/tests)
```

Comment:
my guess is that `ENABLE_PYTHON` should be made an option again in main CMakeLists so that one can build lib _or_ lib+py-intf

Reply:
I don't follow. Are you saying the main CMakeLists used to have an Enable Python option that needs to be restored? And what's `lib+py-intf`?